scripts/mutect2_wdl/mutect2_multi_sample.wdl

```diff
@@ -57,7 +58,8 @@ workflow Mutect2 {
 				dbsnp_index=dbsnp_index,
 				cosmic=cosmic,
 				cosmic_index=cosmic_index,
-				picard_jar=picard_jar
+				picard_jar=picard_jar,
```

Comment:
@davidbenjamin It's not a problem, but now you need GATK4 *and* Picard to make this workflow work.  That was definitely not assumed.  (For example, the gatk-protected docker image has no picard jar).  If you want to use the picard jar, you need to make sure the docker image has it.  

AND the GATK will soon include the picard tools, I believe.  Talk to @droazen or @lbergelson to make sure you are not out-of-date here.

TL;DR
1)  Talk to David or Louis and find out if you can call picard from GATK jar.
2)  If yes, adjust this WDL to do so.
3)  If no, make sure the picard jar is in the docker image.  Make sure to peg it to a specific version.